client/webserver/site/src/js/markets.js

```diff
@@ -568,11 +570,11 @@ export default class MarketsPage extends BasePage {
       updateUserOrderRow(row, order)
       if (order.type === LIMIT) {
         if (order.tif === standingTiF && order.status < statusExecuted) {
-          const icon = row.querySelector('[data-col=cancel] > span')
+          const icon = Doc.tmplElement(row, 'cancelBttn')
           Doc.show(icon)
-          bind(icon, 'click', e => {
+          bind(row, 'click', e => {
             e.stopPropagation()
-            this.showCancel(icon, order.id)
+            this.showCancel(row, order.id)
```

Comment:
Clicking anywhere on the row now opens the cancel form, which does not seem like what we want (esp. if clicking will eventually give order details, etc.)  Can this remain bound to `icon` and still pass `row` to `showCancel`?

Reply:
Another problem with the binding to `row` is that it stays bound, so you can click and get a cancel prompt even when the order is in "cancelling" or "cancelled" status.